configure.ac

```diff
@@ -523,6 +523,7 @@ if test "$WITH_ECONF" = "yes" ; then
   PKG_CHECK_MODULES([ECONF], [libeconf], [],
   [AC_CHECK_LIB([econf],[econf_readDirs],[ECONF_LIBS="-leconf"],[ECONF_LIBS=""])])
   if test -n "$ECONF_LIBS" ; then
+    AC_CHECK_LIB([econf],[econf_readDirsWithCallback], [], [AC_MSG_ERROR([Please update libeconf])])
     ECONF_CFLAGS="-DUSE_ECONF=1 $ECONF_CFLAGS"
   fi
```

Comment:
This configure snippet requires that `libeconf` either doesn't exist or is fresh enough, which is somewhat unusual.
Wouldn't it be better to just check for `econf_readDirsWithCallback` instead?

Reply:
Yes, it is much more simple.